docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,143 @@
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# ParaTime Observer Node
+
+:::info
+
+These instructions are for setting up a _ParaTime observer_ node, which observes ParaTime activity and can issue
+queries that can access confidential data. If you just want to run a _ParaTime client_ node, see the
+[instructions for running a ParaTime client node](paratime-client-node.mdx). If you want to run a _ParaTime_ node
+instead, see the [instructions for running a ParaTime node](paratime-node.mdx). Similarly, if you want to run a
+_validator_ or a _non-validator_ node instead, see the [instructions for running a validator node](validator-node.mdx)
+or [instructions for running a non-validator node](non-validator-node.mdx).
+
+:::
+
+:::tip
+
+The observer node role allows any ParaTime client node with appropriate TEE support to run confidential transactions
+and view calls on Sapphire ParaTime. Observer nodes are client nodes that are registered on chain so that their
+eligibility can be enforced by an on-chain policy (e.g. key manager committees can grant them permissions).
+
+:::
+
+This guide will cover setting up your ParaTime observer node for the Oasis Network. Observer nodes are ParaTime
+client nodes that support confidential queries without being elected into the compute committee. This guide assumes
+some basic knowledge on the use of command line tools.
+
+## Prerequisites
+
+Before following this guide, make sure you've followed the [Prerequisites](prerequisites),
+[Run a Non-validator Node](non-validator-node.mdx), and [Run a ParaTime client Node](paratime-client-node.mdx)
+sections and have a working ParaTime client node with Intel SGX support.
```

Comment:
Link to how to set up the TEE environment. It is better to refer with TEE instead of SGX, because once other TEE solutions become trustworthy we might add support for them too.

Reply:
Linked and replaced Intel SGX with a more general "TEE".